services/slackServices/shoutout.js

```diff
@@ -126,8 +153,83 @@ exports.submitDialog = async reqInfo => {
       title: 'Hurray, You are the newest shoutout Recipient',
       text: `You just received a shoutout from <@${reqInfo.userId}> on <#${org.channel_id}>`,
     };
-
     await slackModel.message.postOpenMessage(recipientAlert);
+
+    const dbInfo = {
+      giver_id: reqInfo.userId,
+      receiver_id: reqInfo.recipient,
+      message: reqInfo.content,
+      organization_id: org.slack_org_id,
+      access_token: org.access_token,
+    };
+
+    await ShoutOutHelper.saveToDatabase(dbInfo);
+  } catch (err) {
+    console.log(err);
```

Comment:
You may have forgotten to remove the `console.log(err);`

Reply:
I have changed this to return null. It's not a part of any HTTP request so I'm not what to do with it as the other slack services follow the same pattern. Let me know how I should proceed.